examples/incito-publication-template.html

```diff
@@ -66,7 +66,16 @@
             data-component-publication-disable-global-scrollbar="false"
             data-translation-keys--publication-viewer-shopping-list-label="Shopping List"
             data-translation-keys--publication-viewer-download-button="Download Now"
-            defer
         ></script>
+
+        <script>
+            (async () => {
+                const publicationData = await SGN.PublicationData
```

Comment:
Missing ;

Reply:
I like it - but could SGN.PublicationData be tweaked? Could maybe be `SGN.component.{{componentName}}`?